kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
Removing source.service seems odd to me. All the nodes in the "Service Mesh" are "services", and one of the common filters we use in rules is to enable/disable something based on what service (often qualified further to some specific version/labels) is making the request. Am I missing something?